webpack.config.js

```diff
@@ -81,4 +81,7 @@ module.exports = {
 	resolve: {
 		extensions: [ ".json", ".jsx", ".js", ".svg" ],
 	},
+	node: {
```

Comment:
If this is necessary, we should investigate another library for retrieving RSS feeds.